### PR TITLE
French Translations

### PR DESCRIPTION
**Pull request by ewanmay**
Actions:
- Made sure all strings were translated
- Added translations we were missing or the ones that were incorrect
- Added a url so you could click on the link for the study pins

### CODE REVIEW

File: src/utils/translate/en.json [hunk 1]
```diff
@@ -2,22 +2,18 @@
   "95%ConfidenceInterval": "95% Confidence Interval",
```
Comment (atatmaja) [inline src/utils/translate/en.json:2]: What's the purpose of leaving this old en.json file?

**Review by atatmaja — COMMENTED** [review]
Looks fine to me once you answer my question

**Comment by atatmaja** [comment]
Oh also we should find a place to commit any of the scripts you used to update the language json files @ewanmay

File: src/utils/translate/en.json (hunk 1 above)
Comment (ewanmay) [reply src/utils/translate/en.json:2]: Because you need it to hold the english strings

**Comment by ewanmay** [comment]
> commit any of the scripts you used to update the language json files

Did not use any scripts aside from the one you used to get things started

**Comment by atatmaja** [comment]
> > commit any of the scripts you used to update the language json files
> 
> Did not use any scripts aside from the one you used to get things started

Did you account for new strings that were added after the translation spreadsheet was made? If so, I guess you went through the spreadsheets manually?

File: src/utils/translate/en.json (hunk 1 above)
Comment (atatmaja) [reply src/utils/translate/en.json:2]: Uhh but we also have another file at path `src/utils/translate/newjson/en.json`, what's the distinction here

Comment (ewanmay) [reply src/utils/translate/en.json:2]: that was not supposed to be there, I have deleted it

**Comment by ewanmay** [comment]
> > > commit any of the scripts you used to update the language json files
> > 
> > 
> > Did not use any scripts aside from the one you used to get things started
> 
> Did you account for new strings that were added after the translation spreadsheet was made? If so, I guess you went through the spreadsheets manually?

Yes I did, I went through them manually

**Review by atatmaja — APPROVED** [review]

**Pull request merged**